app/controllers/business/events_controller.rb

```diff
@@ -57,4 +59,15 @@ def set_event
   def event_params
     params.require(:event_new).permit Event::EVENT_PARAMS
   end
+
+  def load_room
+    @room = Room.find_by id: params[:event_new][:room_id]
+    @event_load = Event.by_room_id params[:event_new][:room_id]
+    @reports = @room.reports.page(params[:page]).per Settings.pagination_commit
+    if params[:event_new][:day].present?
+      @monday = Date.parse(params[:event_new][:day])
+    else
+      @monday = DateTime.now - DateTime.now.cwday + Settings.one
+    end
```

Comment:
Viết như này có được ko e
```suggestion
    @monday = DateTime.now - DateTime.now.cwday + Settings.one
    @monday = Date.parse(params[:event_new][:day]) if params[:event_new][:day].present?
```

Reply:
được ạ